crates/python/test.py

```diff
@@ -1,29 +0,0 @@
-import os
```

Comment:
Was this file intentionally deleted?

Reply:
Yes it was. We used this really early on to test the bindings, it doesn't even reference the right module anymore (`qcs` instead of `qcs_sdk`). #155 will effectively replace this file.